go/common/grpc/policy/watcher.go

```diff
@@ -2,35 +2,83 @@ package policy
 
```

Comment:
This now seems like a bad place to put this -- previously it was a generic policy watcher that worked over gRPC, but now it is specific to sentry nodes. It should move somewhere under `go/sentry`, maybe even `go/sentry/client`?

Reply:
Good idea, I'll move it.